Makefile

```diff
@@ -12,14 +12,16 @@ test:
 	# Create a new project with the default values
 	cookiecutter . --no-input
 
+	virtualenv repo_name/.venv
+
 	# Execute the project's tests
-	cd repo_name && make requirements
-	cd repo_name && python manage.py makemigrations
-	cd repo_name && make migrate
-	cd repo_name && make validate
+	. repo_name/.venv/bin/activate && cd repo_name && make local-requirements
```

Comment:
Annoyingly, `source` isn't something understood by `/bin/sh`, which is what this makefile is using.